qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -769,8 +836,8 @@ def figure(
                 the content of the figure is returned instead.
 
         Returns:
-            The size of the figure if `file_name` is specified. Otherwise the
-            content of the figure in bytes.
+            The size of the figure if `file_name` is specified.
+            Otherwise the `FigureData`.
```

Comment:
```suggestion
            Otherwise the :class:`.FigureData`.
```
Also FigureData should be added to an `__init__` doc tree somewhere. The best spot would probably be in `.framework.__init__` *Experiment Data Classes* list, even if the class def is in this file (like how JobStatus etc are imported there for API docs).

Reply:
Added